python_modules/dagster-graphql/dagster_graphql/schema/asset_selections.py

```diff
@@ -25,5 +27,12 @@ def resolve_assetKeys(self, _graphene_info):
             for asset_key in self._asset_selection.resolve(asset_graph)
         ]
 
+    def resolve_assets(self, graphene_info):
+        asset_graph = ExternalAssetGraph.from_external_repository(self._external_repository)
+        return [
+            get_asset(graphene_info, asset_key)
```

Comment:
i think this is potentially pretty expensive to call in a for loop if there are many assets - we may want a get_assets function that takes in a list of keys

Reply:
I also don't think we need to make the instance.has_asset_key(asset_key) call that that method makes in this context

Reply:
Oof, in that case do you want to handle this, I'll remove my python changes in this PR